components/brave_wallet_ui/panel/container.tsx

```diff
@@ -386,16 +386,20 @@ function Container () {
   if (selectedAccount && (selectedPendingTransaction || signMessageData.length) &&
     selectedPanel === 'connectHardwareWallet') {
     return (
-      <PanelWrapper isLonger={false}>
-        <StyledExtensionWrapper>
-          <ConnectHardwareWalletPanel
-            onCancel={onCancelConnectHardwareWallet}
-            account={selectedAccount}
-            hardwareWalletCode={hardwareWalletCode}
-            onClickInstructions={onClickInstructions}
-          />
-        </StyledExtensionWrapper>
-      </PanelWrapper>
+      <BrowserRouter>
+        <Switch>
```

Comment:
Will this work without the <Switch>?

I also have a PR ready that creates the BrowserRouter in the panel container component, so we should check for regressions when mine lands

Reply:
Removed `switch`